pkg/dashboard/websocket.go

```diff
@@ -3,38 +3,41 @@ package dashboard
 import (
 	"github.com/labstack/echo/v4"
 
+	"github.com/iotaledger/hive.go/core/generics/event"
 	"github.com/iotaledger/hive.go/core/syncutils"
 	"github.com/iotaledger/hive.go/core/websockethub"
 	"github.com/iotaledger/inx-dashboard/pkg/jwt"
 )
 
+type WebSocketMsgType byte
+
 const (
 	// MsgTypeSyncStatus is the type of the SyncStatus message.
-	MsgTypeSyncStatus byte = iota
+	MsgTypeSyncStatus WebSocketMsgType = iota
 	// MsgTypePublicNodeStatus is the type of the PublicNodeStatus message.
-	MsgTypePublicNodeStatus = 1
```

Comment:
maybe add the underlying values to the comments. This makes debugging easier without having to manually count